package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "monaco-components",
+  "type": "module",
   "private": true,
   "scripts": {
     "clean": "npm run clean --workspaces",
     "clean:watch": "tsc --build tsconfig.build.json --clean",
-    "dev": "vite",
-    "dev:debug": "vite --debug --force",
+    "dev": "node --experimental-import-meta-resolve ./node_modules/vite/bin/vite.js",
+    "dev:debug": "node --experimental-import-meta-resolve ./node_modules/vite/bin/vite.js --debug --force",
```

Comment:
I think even with experimental feature enabled it still gives a warning right?

Reply:
No really, this is just required for the copied vite plugin / the import.meta workaround.